tests/upgrade/0.8.2-0.9.0/jiva/volume_test.go

```diff
@@ -72,6 +78,32 @@ var _ = Describe("[jiva] TEST VOLUME UPGRADE", func() {
 			podCount := ops.GetPodRunningCountEventually("default", upgradedLabel, replicaCount+1)
 			Expect(podCount).To(Equal(replicaCount+1), "while checking pod version")
 
+			By("verifying registered replica count and replication factor")
+			podList, err := ops.PodClient.List(metav1.ListOptions{LabelSelector: ctrlLabel})
+			Expect(err).ShouldNot(HaveOccurred(), "while listing controller pod")
+			replicationFactor := podList.Items[0].Spec.Containers[0].Env[0].Value
```

Comment:
It's good to iterate over containers and then over Env, so that later if we change the order of containers, or change the name of Env[0] key, it wouldn't fail.

Reply:
Please add check if the key is "REPLICATION_FACTOR" also.

Reply:
Done